probe-rs/src/core.rs

```diff
@@ -731,6 +626,8 @@ impl<'probe> Core<'probe> {
     /// # Arguments
     ///
     /// * `ranges`: Memory ranges that should be dumped.
+    #[cfg(feature = "debug")]
+    #[cfg_attr(probers_docsrs, doc(cfg(feature = "debug")))]
```

Comment:
I think we might as well invert this into `CoreDump::dump_core`

Reply:
Yeah, I considered moving it completely into the debug module. Might be cleaner.